testing/opencolorio/APKBUILD

```diff
@@ -0,0 +1,77 @@
+# Contributor: Mark Riedesel <mark@klowner.com>
+# Maintainer:
+pkgname=opencolorio
+pkgver=1.0.9
+pkgrel=0
+pkgdesc="A color management framework for visual effects and animation"
+url="http://opencolorio.org"
+arch="all"
+license="BSD"
+depends=""
+makedepends="cmake python2-dev python3-dev freeglut-dev glew-dev lcms2-dev tinyxml-dev yaml-cpp-dev boost-dev"
+install=""
+subpackages="py2-$pkgname:py2 py3-$pkgname:py3 $pkgname-dev $pkgname-utils"
+source="$pkgname-$pkgver.tar.gz::http://github.com/imageworks/OpenColorIO/archive/v$pkgver.tar.gz
+	yaml-cpp-0.5-support.patch"
```

Comment:
Please add also [this patch](https://github.com/imageworks/OpenColorIO/commit/1384029df00525c479e2c7e52fdfa294413fceb6) that fixes dereference of temporary when yaml-cpp 0.5.x is used.